src/Game/UI/Gumps/SpellbookGump.cs

```diff
@@ -1437,6 +1448,68 @@ public override void Update(double totalTime, double frameTime)
                     Hue = 0;
                 }
             }
+
+            /// <summary>
+            /// Overide Draw method to include + icon when ShowEdit is true
+            /// </summary>
+            /// <param name="batcher"></param>
+            /// <param name="x"></param>
+            /// <param name="y"></param>
+            /// <returns></returns>
+            public override bool Draw(UltimaBatcher2D batcher, int x, int y)
+            {
+                base.Draw(batcher, x, y);
+
+                if (ShowEdit)
+                {
+                    ResetHueVector();
+
+                    UOTexture lockTexture = GumpsLoader.Instance.GetTexture(0x1086);
```

Comment:
I don't think this art exists in the standard files (not UOR or latest from EA), may need to search for something that works for most client versions

Reply:
https://github.com/andreakarasho/ClassicUO/blob/d5c3ded99bc3480b30d118ce2b1b49e4dd4b0f2c/src/Game/UI/Gumps/AnchorableGump.cs#L145

this is the lock texture from standard UO

Reply:
We almost need a similar system to the localization system but for art/anim/gump graphic IDs to avoid hard coding. But this graphic should be there on a normal install.